docs/tutorials/notebooks/basic_ot_between_datasets.ipynb

```diff
@@ -0,0 +1,376 @@
+{
```

Comment:
<p>Since this is a low-level tutorial, maybe worth adding a few words on what a geometry is. </p><p> </p><p>Essentially this geometry is just two (or one) point clouds compared with a <code>SqEuclidean</code> cost between points here. Maybe this can be discussed by setting <code>cost_fn</code> to None (the default) and mention this defaults to <code>SqEuclidean</code> (linked in doc).</p><p> </p><p>Maybe also mention that <code>epsilon</code> regularization in Sinkhorn is also stored in the geometry (and set by default to 1/20th of mean cost value). This can come later when running Sinkhorn (e.g. a comment  "Wait? Sinkhorn? but where's <code>epsilon</code> ? ) </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ott-jax/ott/pull/296/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>